.github/workflows/ci.yml

```diff
@@ -0,0 +1,16 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  # Every day at 3:00pm UTC
+  schedule:
+    - cron: '0 15 * * *'
+
+jobs:
+  ci:
+    name: CI
+    # Only run cron on the silverstripe account
+    if: (github.event_name == 'schedule' && startsWith(github.repository, 'silverstripe/')) || (github.event_name != 'schedule')
+    uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
```

Comment:
Do we need the slack integration defined in the travis config (do we know if that was even still doing anything)?

Reply:
No